lib/util/util.mk

```diff
@@ -1 +1 @@
-UTIL_OBJS := params.o logging.o util.o stats.o
+UTIL_OBJS := params.o logging.o util.o stats.o xpcapng.o
```

Comment:
Why the 'x' prefix in the file name?

Reply:
x was for xdp-tools, I can remove it if you prefer plain pcapng.c/h

Reply:
Right, I see. Hmm, I kinda see the point with that, so I guess we can keep it. I just thought it was odd because 'xpcapng' looks a lot like someone misspelled 'xdp' :)